test/end-to-end/pages/deceased/address.js

```diff
@@ -22,6 +18,7 @@ async function enterDeceasedAddressManually(language ='en', testAddressIndex) {
     await I.waitForVisible('#postcodeAddress');
     await I.selectOption('#postcodeAddress', testAddressIndex);
     await I.waitForElement('#addressLine1');
+    await I.wait(2);
     await I.navByClick(commonContent.saveAndContinue);
```

Comment:
do we need this? Can we use an explicit wait rather than an implicit hard-coded wait? I did spend quite a long time taking out hard-coded waits so would only like to see them come back in if there's no alternative with an explicit wait! If we need an explicit wait can we lower it? wait method will take decimal values. Also, did the issue arise with auto-delay disabled? In some places I have a put a small wait of 0.25s as I just couldn't find a way with explicit waits, but only if auto delay switched off.

Reply:
good call @vinceholt - I've removed the implicit wait for a better waitForText() which looks to work.